pkg/workloads/console/runner/runner.go

```diff
@@ -736,10 +769,56 @@ func (c *Runner) GetAttachablePod(csl *workloadsv1alpha1.Console) (*corev1.Pod,
 	}
 
 	for _, c := range pod.Spec.Containers {
-		if c.TTY {
+		if !csl.Spec.Interactive || c.TTY {
 			return pod, c.Name, nil
 		}
 	}
 
 	return nil, "", errors.New("no attachable pod found")
 }
+
+// NoninteractiveAttacher knows how to attach to stdio of an existing container, relaying io
+// to the parent process file descriptors.
```

Comment:
This comment should probably be expanded on - it would be useful to lay out the differences between this and the interactive version

Reply:
I think this is probably just copypasta.

Reply:
Just spotted that similarly, the comment above `func (a *NoninteractiveAttacher) Attach` is wrong

Reply:
And worst of all, it has a missing apostrophe 😱